src/App.css

```diff
@@ -477,21 +316,12 @@ a.text-link {
 .toolbar-icon {
   cursor: pointer;
   display: inline-block;
-  padding: 5px 8px;
+  padding: 15px 8px 5px 8px;
```

Comment:
I removed the descendant selector below, and added the padding directly to the toolbar-icon.
```css
.toolbar-icon .icon {
  padding-top: 10px;
  flex: 1;
}
```

Reply:
This is why I could remove the inline paddingTop from sentence splitter, however now I realize I actually still may need to add 2px, since sentence splitter had `paddingTop: 8px` instead of 10px.
<img width="615" alt="Screenshot 2024-08-28 at 3 11 51 PM" src="https://github.com/user-attachments/assets/c236a986-56b5-4aeb-a980-c883161339cd">

Reply:
Got it, thanks!